src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -35,62 +59,66 @@
+(defview passphrase-display [passphrase]
+  [react/text  {:style (styles/recovery-phrase-input true)} passphrase])
 
-(defview recover []
+(defview recoverA [cofx]
   (letsubs [recovered-account [:get-recover-account]
             node-status? [:node-status]]
     (let [{:keys [passphrase password passphrase-valid? password-valid?
                   password-error passphrase-error passphrase-warning processing?]} recovered-account
           node-stopped? (or (nil? node-status?)
                             (= :stopped node-status?))
-          valid-form? (and password-valid? passphrase-valid?)
+          valid-form? (and passphrase-valid?)
           disabled?   (or (not recovered-account)
                           processing?
                           (not valid-form?)
                           (not node-stopped?))
-          sign-in     #(re-frame/dispatch [:accounts.recover.ui/sign-in-button-pressed])]
+          sign-in     #(re-frame/dispatch [:accounts.recover.ui/recover-account-confirmed])]
       [react/keyboard-avoiding-view {:style styles/screen-container}
-       [status-bar/status-bar]
-       [toolbar/toolbar nil toolbar/default-nav-back
-        [toolbar/content-title (i18n/label :t/sign-in-to-another)]]
-       [react/view styles/inputs-container
-        [passphrase-input (or passphrase "") passphrase-error passphrase-warning]
-        [password-input (or password "") password-error (when-not disabled? sign-in)]
-        (when platform/desktop?
-          [react/i18n-text {:style styles/recover-release-warning
-                            :key   :recover-account-warning}])]
+       [cancel-link]
+       [step-i-of-n 1 2]
+       [recovery-phrase-title]
+       [recovery-phrase-explainer]
+       (if processing?
+         [passphrase-display passphrase]
+         [passphrase-input (or passphrase "") passphrase-error passphrase-warning])
+       (if (not (string/blank? (cofx :error))) [react/i18n-text {:style styles/recovery-error-message
+                                                                 :key   :recovery-error-message}])
+       (when platform/desktop?
+         [react/i18n-text {:style styles/recover-release-warning
+                           :key   :recover-account-warning}])
        [react/view components.styles/flex]
        (if processing?
          [react/view styles/processing-view
           [react/activity-indicator {:animating true}]
           [react/i18n-text {:style styles/sign-you-in
-                            :key   :sign-you-in}]]
+                            :key   :processing}]]
          [react/view {:style styles/bottom-button-container}
           [react/view {:style components.styles/flex}]
           [components.common/bottom-button
            {:forward?  true
-            :label     (i18n/label :t/sign-in)
+            :label     (i18n/label :t/next)
             :disabled? disabled?
             :on-press  sign-in}]])])))
+
+(defview recoverB [cofx]
```

Comment:
Same here: we should not pass `cofx` to a view.

Reply:
Done!